### PR TITLE
Feature/Implement schedule section

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyen-agilityio @hungnguyenhuu Please help me to check PR and merge it.

### CODE REVIEW

File: practice-two/src/assets/css/pages/home/schedule.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.schedule {
+  margin-top: 50px;
+}
+
+.schedule__wrapper {
+  width: 100%;
+  background: var(--dodger-blue-color);
+  position: relative;
+}
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/schedule.css:9]: Maybe you need to check the order of priority, right?

File: practice-two/src/assets/css/pages/home/schedule.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+.schedule__description {
+  margin-top: 50px;
+  margin-bottom: 40px;
+  color: var(--light-color);
+  font-weight: var(--fw-normal);
+  line-height: 40px;
+}
+
+.schedule__image {
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/schedule.css:34]: naming

Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/schedule.css:34]: do you mean the ` schedule__image`  is not good?

File: practice-two/src/assets/css/pages/home/schedule.css (hunk 1 above)
Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/schedule.css:9]: i have updated it. please help me to check it again @hungnguyenhuu

File: practice-two/src/assets/css/pages/home/schedule.css (hunk 2 above)
Comment (hungnguyenhuu) [reply practice-two/src/assets/css/pages/home/schedule.css:34]: you set the display none but your naming don't show that, right?

**Pull request merged**